sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/spice_indexer.py

```diff
@@ -285,23 +286,22 @@ def write_data_to_efs(s3_key: str, s3_bucket: str, spice_mount_path: Path) -> Pa
     # Create an S3 client
     s3_client = boto3.client("s3")
 
-    # Keep the base folder name and filename from the s3 key
-    # i.e. "ck/file.bc"
+    # Get directory structure from the S3 key
     dirname, filename = os.path.split(s3_key)
-    s3_folder_path = os.path.basename(dirname)
-    # Download path to EFS
-    efs_spice_path = spice_mount_path / s3_folder_path
-    efs_spice_filename_and_path = efs_spice_path / filename
+    # Prepend EFS path to the s3 directory structure
+    efs_spice_path = spice_mount_path / dirname
```

Comment:
Do you want to call this "data" mount path now since it is the generic data folder instead of spice specific folder?
```suggestion
    efs_spice_path = data_mount_path / dirname
```